model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -352,6 +356,20 @@ message ParDoPayload {
   bool splittable = 6;
 }
 
+// The payload for the primitive SplittableProcessKeyed transform.
+message SplittableProcessKeyedPayload {
```

Comment:
Did you/we ever separate the non-primitive payloads into another file? This should go there.

Reply:
No, this hasn't happened.